packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/import/saved_objects_importer.ts

```diff
@@ -75,7 +84,19 @@ export class SavedObjectsImporter implements ISavedObjectsImporter {
       typeRegistry: this.#typeRegistry,
       importHooks: this.#importHooks,
       managed,
-    });
+    })
+      .then((response: SavedObjectsImportResponse) => {
+        this.#log.info(`🌸 Successfully imported ${response.successCount} saved objects.`);
+        return response;
+      })
+      .catch((error) => {
+        this.#log.error('Failed to import saved objects');
```

Comment:
Log the errors from within `importSavedObjectsFromStream` [before](https://github.com/elastic/kibana/blob/d94bf9a41b264fd0fd268d20317dc27f5678c2ae/packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/import/import_saved_objects.ts#L187-L195) the `return`